main/EDTypesMod.F90

```diff
@@ -410,7 +410,7 @@ module EDTypesMod
      integer  ::  ncl_p                                            ! Number of occupied canopy layers
      integer  ::  anthro_disturbance_label                         ! patch label for anthropogenic disturbance classification
      real(r8) ::  age_since_anthro_disturbance                     ! average age for secondary forest since last anthropogenic disturbance
-
+     integer  ::  nocomp_pft_label                                 ! where nocomp is active, use this label for patch ID.   
```

Comment:
add text here that describes the special case where = 0
Also, short description of how labels are determined.